src/main/java/com/easypost/easyvcr/Censors.java

```diff
@@ -155,12 +153,17 @@ public String applyBodyParametersCensors(String body) {
      * @param headers Map of headers to apply censors to.
      * @return Censored map of headers.
      */
-    public Map<String, List<String>> applyHeadersCensors(Map<String, List<String>> headers) {
+    public Map<String, List<String>> censorHeaders(Map<String, List<String>> headers) {
         if (headers == null || headers.size() == 0) {
             // short circuit if there are no headers to censor
             return headers;
         }
 
+        if (headersToCensor.size() == 0) {
+            // short circuit if there are no censors to apply
+            return headers;
+        }
```

Comment:
```suggestion
        if (headers == null || headers.size() == 0 || headersToCensor.size() == 0) {
            // short circuit if there are no headers to censor
            return headers;
        }
```

we can combine these two, won't block it